src/data/admin/statistics.ts

```diff
@@ -0,0 +1,20 @@
+type StatisticsData = {
+  title: string
+  quantity: number
+}
+
+export const getStatistics = async () => {
+  const response = await fetch('http://localhost:3000/api/admin/statistics', {
```

Comment:
tu jakimiś envami trzeba baseUrl podać, bo nie będzie na produkcji to działać :/